Assets/PatchKit Patcher/Scripts/AppData/FileSystem/DirectoryOperations.cs

```diff
@@ -21,8 +23,8 @@ public static class DirectoryOperations
         /// <exception cref="UnauthorizedAccessException">Unauthorized access.</exception>
         public static bool IsDirectoryEmpty(string dirPath)
         {
-            Checks.ArgumentNotNullOrEmpty(dirPath, "dirPath");
-            Checks.DirectoryExists(dirPath);
+            Assert.IsFalse(string.IsNullOrEmpty(dirPath));
```

Comment:
Why change to assertions here and other places?

Reply:
The methods in `Checks` class are deprecated. I figured I might as well, remove them.

Reply:
They are deprecated with comment "Do custom checks and manual exception throwing instead.". 

It means that they should be replaced with manual checking like

```
if(string.IsNullOrEmpty(dirPath))
{
  throw new ArgumentException("Value cannot be null or empty", "dirPath");
}
```

Assertions are made for completely different thing, definitively not checking whether API (public) call has been done with correct arguments.